arctic/_cache.py

```diff
@@ -3,43 +3,79 @@
 
 from pymongo.errors import OperationFailure
 
-from ._config import CACHE_COLL, CACHE_DB
-
 logger = logging.getLogger(__name__)
 
+CACHE_COLL = 'cache'
+CACHE_DB = 'meta_db'
+CACHE_SETTINGS = 'cache_settings'
+DEFAULT_CACHE_EXPIRY = 3600
+
 
 class Cache:
```

Comment:
I suggest showing the config entries as an example here.  So it's clear that enabled is a key etc.

Reply:
Done.